source/jquery.shinybox.js

```diff
@@ -202,13 +202,8 @@
 
                 if ("onorientationchange" in window) {
                     var calculateWidthAndHeight = function () {
-                        if (window.orientation === 0) {
-                            width = winWidth;
-                            height = winHeight;
-                        } else if (window.orientation === 90 || window.orientation === -90) {
-                            width = winHeight;
```

Comment:
Hi Maarten,

I have tested it in both Android and IOS devices and it works. When the
device rotates, it triggers resize event in which the new windows height
and and new windows width is taken again from window.innerHeight.
Please do let me know if you have faced any issue in devices while rotating
with this new code.

Thanks,
Sathish

On Thu, Mar 17, 2016 at 4:47 PM, Maarten Winter <notifications@github.com>
wrote:

> In source/jquery.shinybox.js
> <https://github.com/One-com/shinybox/pull/24#discussion_r56497579>:
>
> > @@ -202,13 +202,8 @@
> >
> >                  if ("onorientationchange" in window) {
> >                      var calculateWidthAndHeight = function () {
> > -                        if (window.orientation === 0) {
> > -                            width = winWidth;
> > -                            height = winHeight;
> > -                        } else if (window.orientation === 90 || window.orientation === -90) {
> > -                            width = winHeight;
>
> Hey @sathishkategaru <https://github.com/sathishkategaru>, here you no
> longer react to the orientation of 90 or -90 degrees by using the winHeight
> as width, and winWidth as height, which I think will cause issues when you
> rotate the device.
> From the PR description I can't see if you tested device orientation or
> not, could you check that? If that does still work, could you explain
> exactly *how* it calculates the correct width/height when the device is
> rotated?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/One-com/shinybox/pull/24/files/33716d81c0b73563c787ccada5c51cc8b62c40a0#r56497579>
>